lib/api_remote.ml

```diff
@@ -36,7 +35,7 @@ module Github : Api.Github = struct
       | "base64" ->
         begin
           try
-            response.content |> String.split_lines |> String.concat |> decode_string_pad |> Config_j.config_of_string
+            response.content |> String.split_on_char '\n' |> String.concat "" |> decode_string_pad |> Config_j.config_of_string
```

Comment:
`String.split_on_char '\n' |> String.concat ""` this is essentially removing all `\n` in the string, but this implementation allocates 2x the size of the original string. Consider using regex, e.g. `Re2.replace`.